src/annotator/anchoring/range.js

```diff
@@ -423,17 +440,15 @@ export class SerializedRange {
     // Node.compareDocumentPosition() to decide when to set the
     // commonAncestorContainer and bail out.
 
-    const contains = (a, b) => a.compareDocumentPosition(b) & 16;
-    $(range.startContainer)
-      .parents()
-      .each(function () {
-        if (contains(this, range.endContainer)) {
-          range.commonAncestorContainer = this;
-          // bail out of loop
-          return false;
-        }
-        return true;
-      });
+    const contains = (a, b) =>
```

Comment:
The comment above still likely applies, but does reference jquery so we should probably at least change that or delete the parts of it that don't make sense anymore.

Reply:
Agreed. I re-tested the test case given in the comment above and was no longer able to reproduce. I couldn't find any relevant bug reports in WebKit's bug tracker either. On that basis I simplified this code with `Node.contains(...)`.

In future I suspect we might be able to remove `Range` normalization from most of the code. The only place where it is really needed AFAIK is before wrapping nodes in highlights.